### ISSUE TITLE
Failed to load the Termrk package

### ISSUE THREAD

**Issue by Lageos** [romgrk/termrk#47]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.0.5
**System**: Linux Mint
**Thrown From**: [Termrk](https://github.com/romgrk/termrk) package, v0.1.22
### Stack Trace

Failed to load the Termrk package after install.

```
At error evaluating function `fade`: undefined is not a function in /home/michael/.atom/packages/Termrk/styles/termrk.less:23:20

LessError: error evaluating function `fade`: undefined is not a function
  at /home/michael/.atom/packages/Termrk/styles/termrk.less:23:20
```
### Commands

```
     -1:51.2.0 core:move-up (div.terminal)
     -1:50.9.0 core:confirm (div.terminal)
     -1:49.6.0 core:move-up (atom-workspace.workspace.scrollbars-visible-always.theme-atom-material-syntax.theme-atom-soda-dark-ui)
     -1:49.2.0 core:confirm (div.terminal)
  8x -1:47.3.0 core:backspace (div.terminal)
     -1:43.1.0 core:confirm (div.terminal)
  2x -1:42 core:move-up (atom-workspace.workspace.scrollbars-visible-always.theme-atom-material-syntax.theme-atom-soda-dark-ui)
     -1:41 core:confirm (div.terminal)
     -1:29.3.0 core:move-up (div.terminal)
     -1:28.3.0 core:confirm (div.terminal)
     -1:27.6.0 core:move-up (atom-workspace.workspace.scrollbars-visible-always.theme-atom-material-syntax.theme-atom-soda-dark-ui)
     -1:27.3.0 core:confirm (div.terminal)
     -1:26.9.0 core:move-up (atom-workspace.workspace.scrollbars-visible-always.theme-atom-material-syntax.theme-atom-soda-dark-ui)
     -1:26.6.0 core:confirm (div.terminal)
     -0:16.9.0 core:paste (atom-text-editor.editor.mini.is-focused)
     -0:16.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "atom-runner"
    ],
    "themes": [
      "atom-soda-dark-ui",
      "atom-material-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.28.10
atom-material-syntax, v0.2.1
atom-soda-dark-ui, v1.1.0
file-icons, v1.6.2
language-ansys, v0.0.0
language-latex, v0.6.1
minimap, v4.12.2
script, v2.26.3
term2, v0.9.14

# Dev
No dev packages
```

**Comment by romgrk** [comment]
Fixed by 08a7c692907cedaf7cbdec61342523f4c683c1a7. This is a weird one, you seemed to be the only one to have problem compiling the stylesheet.

**Comment by ghost** [comment]
I'm having the same issue #56
